### PR TITLE
UI updates for who we are 

### PR DESCRIPTION
**Pull request by Akshaya284**
Fix # https://adobe-acs-group.atlassian.net/browse/WRLDBAEMS-82

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/en/who-we-are
- After: https://dev-who-we-are--world-bank--aemsites.hlx.live/en/who-we-are

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -421,6 +421,28 @@ main .section.bg-primary-blue-90[data-bg-inside-container="true"] > div {
...
+main .section.sub-header-container > div > p {
+  color: #181f25;
+  font-size: 24px;
+  font-weight: 400;
+  margin: 0.2em 0 1.5em 0;
+}
+
+@media (max-width: 767px) {
+  main .section.sub-header-container > div > p {
+    font-size: 18px !important;
```
Comment (thomsebastin) [inline styles/styles.css:441]: Please don't use `!important`. Try increasing the specificity.

Comment (thomsebastin) [inline styles/styles.css:441]: Use variables from `styles.css`.

File: styles/styles.css [hunk 2]
```diff
@@ -421,6 +421,28 @@ main .section.bg-primary-blue-90[data-bg-inside-container="true"] > div {
...
+main .section.sub-header-container > div > h2 {
+  color: #012740;
+  font-weight: 700;
+  font-size: 36px;
+  margin: 0;
+}
+
+main .section.sub-header-container > div > p {
+  color: #181f25;
+  font-size: 24px;
```
Comment (thomsebastin) [inline styles/styles.css:434]: use variables.

File: blocks/columns/columns.js [hunk 3]
```diff
@@ -2,6 +2,25 @@ export default function decorate(block) {
   const cols = [...block.firstElementChild.children];
   block.classList.add(`columns-${cols.length}-cols`);
 
+  // specific logic for columns-divider-variant
+  if (block.classList.contains('columns-divider-variant')) {
+    block.style.display = 'flex';
+    block.style.gap = '20px';
```
Comment (thomsebastin) [inline blocks/columns/columns.js:8]: Can we do this directly in `CSS` file?

File: blocks/columns/columns.js [hunk 4]
```diff
@@ -2,6 +2,25 @@ export default function decorate(block) {
...
+      } else {
+        col.style.flexBasis = '70%';
+      }
+    });
+  } else {
+    // Default to equal column width
+    cols.forEach((col) => {
+      col.style.flexBasis = `${100 / cols.length}%`;
+    });
+  }
```
Comment (thomsebastin) [inline blocks/columns/columns.js:22]: I think this can be handled via CSS. Let me know if you need to connect.

File: blocks/columns/columns.css [hunk 5]
```diff
@@ -22,12 +22,85 @@
...
+    flex-basis: 100%;
+    border-left: 1px solid #E7EDF3;
+    padding-left: 10px;
+  }
+  
+  .columns.columns-divider-variant > div > div:first-child > p {
+    font-size: 24px;
+    text-align: left;
+    color: #181F25;
+    line-height: 28.8px;
```
Comment (thomsebastin) [inline blocks/columns/columns.css:104]: Let's round the line-height.

File: blocks/columns/columns.css [hunk 6]
```diff
@@ -22,12 +22,85 @@
...
+  }
+  
+  .columns.columns-divider-variant > div > div:last-child p {
+    font-size: 20px;
+  }
+
+  .columns.columns-divider-variant > div > div:first-child > p {
+    font-size: 24px;
+    text-align: left;
+    color: #181F25;
```
Comment (thomsebastin) [inline blocks/columns/columns.css:81]: If you use nesting in CSS most of this can be cleaned up.

File: blocks/columns/columns.css [hunk 7]
```diff
@@ -22,12 +22,85 @@
...
+  }
+}
+
+@media (max-width: 767px) {
```
Comment (thomsebastin) [inline blocks/columns/columns.css:85]: Use media range notation.

**Comment by thomsebastin** [comment]
@Akshaya284 Keep PR in drafts state if there are lint issues.

**Review by mkbansal1 — CHANGES_REQUESTED** [review]
@Akshaya284  @thomsebastin 
I suggest the following:
1. Add a dropdown in columns block "Columns Layout" with options
- Default
- 2 Column (50%,50%)
- 2 Column (33%, 66%)
- 2 Column (66%, 33%)
- 2 Column (25%, 75%)
- 2 Column (75%, 25%)
As this will be generic layout option, we can use it with any variation
2. Add a style variation 
- Vertical bar on 2nd column

No Javascript handling needed for above, just use CSS.

**Comment by Akshaya284** [comment]
> @Akshaya284 Keep PR in drafts state if there are lint issues.

Noted.

**Comment by thomsebastin** [comment]
@mkbansal1 Will handle that in a separate PR. Thanks.

File: blocks/columns/columns.css [hunk 8]
```diff
@@ -19,16 +15,96 @@
...
+  > div:last-child {
+    flex-basis: 66%;
+    border-left: 1px solid var(--neutrals-20);
+    padding-left: 20px;
+    color: var(--neutrals-100);
+    font-size: var(--body-font-size);
+  }
+}
+
+@media (width >= 900px) {
```
Comment (thomsebastin) [inline blocks/columns/columns.css:47]: Update this media query.

File: blocks/columns/columns.css [hunk 9]
```diff
@@ -19,16 +15,96 @@
...
     gap: 32px;
+
+    > div {
+      flex: 1;
+      order: unset;
+    }
   }
+}
+
+@media (width >= 768px) and (width <= 1024px) {
```
Comment (thomsebastin) [inline blocks/columns/columns.css:60]: Can we do `media (width <= 1024px)` and then below `media(width <= 768px)` separately?

**Review by thomsebastin — CHANGES_REQUESTED** [review]

**Review by thomsebastin — APPROVED** [review]

**Review by mkbansal1 — APPROVED** [review]

**Pull request merged**